### DISCUSSION TITLE
How to connect YouTube to AxelChat?

### DISCUSSION THREAD

**Discussion by alchem1no** [3dproger/AxelChat#497] [Q&A] [answered]
What link should I paste to setup YouTube chat?

**Comment by 3dproger** [comment, marked as answer]
You need to paste a link to your broadcast. Here are some examples:
`https://www.youtube.com/watch?v=jfKfPfyJRdk`
`youtu.be/jfKfPfyJRdk`
`youtube.com/video/jfKfPfyJRdk`
`youtube.com/live/jfKfPfyJRdk`
